german/Oktoberfest/oktoberfest.md

```diff
@@ -3,4 +3,6 @@ Sie ist blond und hat lange geflochtene Haare. Als Sie sie anblickt, sie merkt d
 
 [Laden Sie sie zum etwas zu trinken ein](../Einladen/einladen.md)
 
-[Fragen Sie sie "Wie spaet ist es?"](../Fragen/fragen.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.